funky-flamingos/rinky_dink/chat/signals.py

```diff
@@ -0,0 +1,15 @@
+# from django.db.models.signals import post_save
+# from django.contrib.auth.models import User
+# from django.dispatch import receiver
+# from .models import Chat
+
+
+# @receiver(post_save, sender=User)
+# def create_profile(sender, instance, created, **kwargs):
+#     if created:
+#         Chat.objects.create(recipent=instance)
+
+
+# @receiver(post_save, sender=User)
+# def save_profile(sender, instance, **kwargs):
+#     instance.chat.save()
```

Comment:
The plan initially was to have a `chat` model which would contain all the `messages` and a recipent User object. This file was supposed to create a chat object whenever a new User was created.
But the deadline was only less than a day away then and the code was giving errors. Again, the whole chat frontend was left to be completed, so we decided to just make a public chat room available to all users since it would be simpler and quicker to make. I forgot to delete this file before submitting 😅